src/routes/resources/teams/+page.svelte

```diff
@@ -3,37 +3,36 @@
 </script>
 
 <div>
-	<Heading tag="h1" class="mb-4" customSize="text-4xl font-extrabold  md:text-5xl lg:text-5xl"
+	<Heading tag="h1" class="mb-4" customSize="text-4xl font-extrabold  md:text-5xl lg:text-5xl text-purple-600"
```

Comment:
Don't change the header color. in the future if you want to change text color or any other style attribute you need to put it in the class string